src/2sv/Change.vue

```diff
@@ -27,24 +27,30 @@
 </template>
 
 <script>
-import { find, remove, mfa } from '@/global/mfa';
+import { find, remove, removeWebauthn, mfa, retrieve } from '@/global/mfa'
 
 export default {
   data: () => ({
     requested: {},
   isLastOne: mfa.numVerified === 1
   }),
   computed: {
-    label: vm => vm.requested && vm.requested.label || '2SV'
+    label: vm => vm.requested?.label || '2SV',
   },
   methods: {
     async yes(id) {
-      await remove(id)
+      if (mfa.keys.id === id) {
+        await removeWebauthn(mfa.keys.data[0].id, id)
```

Comment:
This doesn't look right to me.
Is the order of the parameters right?
Is the hard coded `0` index right?

Reply:
The order does look incorrect here, but if the user will only "change" keys if they only have 1 left so that's why I hard coded it.